assets/css/style.css

```diff
@@ -422,7 +444,7 @@ input {
     flex-direction: row-reverse;
   }
 
-  .article-btn:hover {
+  .article-btn:active {
```

Comment:
- [ ] Good job making the changes @WilfriedM01, Although I notice you have replaced the `:hover` effect with `:active` effect, Please remember we need both effects. Besides that, the changes are only reflected in the desktop design If you notice on the Figma design these changes are part of the mobile design too. Please consider applying the changes in both layouts.